app-react/src/jsMain/kotlin/com/ustadmobile/mui/components/SocialWarningListItem.kt

```diff
@@ -0,0 +1,76 @@
+package com.ustadmobile.mui.components
+
+import com.ustadmobile.core.MR
+import com.ustadmobile.core.hooks.useStringProvider
+import js.objects.jso
+import mui.material.Box
+import mui.icons.material.Warning as WarningIcon
+import mui.material.Button
+import mui.material.ButtonVariant
+import mui.material.ListItem
+import mui.material.ListItemIcon
+import mui.material.ListItemText
+import react.FC
+import react.Props
+import react.ReactNode
+import web.cssom.AlignItems
+import web.cssom.Display
+import web.cssom.FlexDirection
+import web.cssom.JustifyContent
+import web.cssom.px
+import web.window.WindowTarget
+import web.window.window
+
+external interface SocialWarningProps : Props {
+    var onDismiss: () -> Unit
+}
+
+val SocialWarningListItem = FC<SocialWarningProps> { props ->
+
+    val strings = useStringProvider()
+
+    ListItem {
+        ListItemIcon {
+            WarningIcon()
+        }
+
+        Box {
+            sx = jso {
+                display = Display.flex
+                flexDirection = FlexDirection.column
+                alignItems = AlignItems.start
+            }
+
+            ListItemText {
+                primary = ReactNode(strings[MR.strings.be_careful_interacting_online])
+                secondary = ReactNode(strings[MR.strings.be_careful_not_to_share])
```

Comment:
The buttons are part of the ListItem and should be part of secondary text. The same is done in ClazzDetailOverviewCourseBlockListItem in app-react.

 ```
e.g.
ListItem {
    ListItemText {
      secondary = Stack.create {
         direction = responsive(StackDirection.column)
         + strings[MR.strings.text_constant]
       
         Stack {
            direction = responsive(StackDirection.row)
            //Buttons can go here   
          }
      }
   
     secondaryTypographyProps = jso {
        component = div
     }
   }  
}
```